.github/workflows/integration.yml

```diff
@@ -19,27 +19,26 @@ jobs:
             TestEthereumMultipartyTokensRemoteNameE2ESuite,
             TestFabricGatewayE2ESuite,
             TestFabricMultipartyE2ESuite,
+            TestFabricMultipartyGlobalListenerE2ESuite,
```

Comment:
Because the Fabric version of this test had to be its own suite, I've only added it to the nightly integration (vs. every PR). That also means the first time it will run is the night _after_ this PR gets merged.

Open to thoughts, but we have a lot of jobs on each PR already, and I didn't necessarily want to add another one unless we feel like the coverage is needed.